workflow/rules/picard.smk

```diff
@@ -0,0 +1,36 @@
+__author__ = "Jonas Almlöf"
+__copyright__ = "Copyright 2021, Jonas Almlöf"
+__email__ = "jonas.almlof@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule picard_mark_duplicates:
+    input:
+        "alignment/samtools_extract_reads/{sample}_{type}_{chr}.bam",
+    output:
+        bam=temp("alignment/picard_mark_duplicates/{sample}_{type}_{chr}.bam"),
+        metrics=temp("alignment/picard_mark_duplicates/{sample}_{type}_{chr}.metrics.txt"),
+    params:
+        extra=config.get("picard_mark_duplicates", {}).get("extra", ""),
+    log:
+        "alignment/picard_mark_duplicates/{sample}_{type}_{chr}.bam.log",
+    benchmark:
+        repeat(
+            "alignment/picard_mark_duplicates/{sample}_{type}_{chr}.bam.benchmark.tsv",
+            config.get("picard_mark_duplicates", {}).get("benchmark_repeats", 1),
+        )
+    threads: config.get("picard_mark_duplicates", {}).get("threads", config["default_resources"]["threads"])
+    resources:
+        threads=config.get("picard_mark_duplicates", {}).get("threads", config["default_resources"]["threads"]),
```

Comment:
sort alphabetically, guess the other rules also have threads and time first...

Reply:
done